static/css/style.css

```diff
@@ -14,6 +14,9 @@
     --white: #FFFFFF;
     --black: #000000;
     --bg-grey: #F5F5F5;
+    --img-highlighter: linear-gradient(to bottom, rgba(0,0,0,0) 0%, rgba(0,0,0,0.6) 100%);
+    --contact-bg: linear-gradient(48deg, rgb(229, 71, 37) 50%, rgb(255, 49, 134));
+    --star-rating: #f2d230
```

Comment:
Missing `;` semicolon at end of the line.

Reply:
Added missing semicolon.